### ISSUE TITLE
ImageGrab Grab

### ISSUE THREAD

**Issue by Sangeli** [python-pillow/Pillow#866]
I'm using Windows 8.1 on a 3200x1800 screen. When I use ImageGrab grab method I only get a 1600x900 image which shows only the upper left hand corner of the screen. When I change the resolution to something lower (say 1920 by 1080) it shows up as expected with a full size image. I presume this is due to the fact that 3200x1800 screens are newer and a lot of applications having pixels scaled by a factor of 2 but I'm wondering if there is anything I can do to get screenshots with that resolution with PIL or another module if PIL can't support.

**Comment by Toruitas** [comment]
How exactly did you hardcode this? Within Python or C++? I'm running into the same problem but have never worked with C/++ before. Not sure how to manipulate the device caps.

**Comment by Sangeli** [comment]
I did this in Python. Before taking screenshot I set these variables. I
hard-coded width and height since I only have this designed to function on
my laptop for now.:

hwin = win32gui.GetDesktopWindow()
hwindc = win32gui.GetWindowDC(hwin)
self.srcdc = win32ui.CreateDCFromHandle(hwindc)
self.memdc = self.srcdc.CreateCompatibleDC()
self.width = 3200
self.height = 1800

Then to take the screenshot I ran these lines of code each time:

bmp = win32ui.CreateBitmap()
bmp.CreateCompatibleBitmap(self.srcdc, self.width, self.height)
self.memdc.SelectObject(bmp)
self.memdc.BitBlt((0, 0), (self.width, self.height), self.srcdc, (0, 0),
win32con.SRCCOPY)
bmpstr = bmp.GetBitmapBits(True)
im = Image.frombuffer('RGB', (self.width, self.height), bmpstr, 'raw',
'BGRX', 0, 1)
return im

Hope this helps.
